From c0faa221bfb713b2287ed2f101285d56f28ba091 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 29 Jun 2019 12:56:02 +0800
Subject: [PATCH] do not support network setting for now

We have a long term plan to support it, but not now.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Refactor for anaconda 34.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for anaconda 37.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/core/configuration/system.py | 3 ++-
 pyanaconda/installation.py              | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/pyanaconda/core/configuration/system.py b/pyanaconda/core/configuration/system.py
index 63fb8afc5d..912e7452e9 100644
--- a/pyanaconda/core/configuration/system.py
+++ b/pyanaconda/core/configuration/system.py
@@ -145,7 +145,8 @@ class SystemSection(Section):
     @property
     def can_configure_network(self):
         """Can we configure the network?"""
-        return self._is_boot_iso or self._is_booted_os
+        # Do not support network config for now
+        return False
 
     @property
     def can_detect_enabled_smt(self):
diff --git a/pyanaconda/installation.py b/pyanaconda/installation.py
index b98425e3cc..dff213b09d 100644
--- a/pyanaconda/installation.py
+++ b/pyanaconda/installation.py
@@ -154,7 +154,7 @@ class RunInstallationTask(InstallationTask):
         configuration_queue.append(os_config)
 
         # schedule network configuration (if required)
-        if conf.target.can_configure_network and conf.system.provides_network_config:
+        if False:
             overwrite = payload.type in PAYLOAD_LIVE_TYPES
             network_config = TaskQueue(
                 "Network configuration",
-- 
2.7.4

